api/public_api_v1.yaml

```diff
@@ -144,6 +144,8 @@ paths:
           description: Unauthorized
         '404':
           description: No service found for the provided ID.
+        '429':
+          description: Too many mequests
```

Comment:
typo (this states for every othre dscription in thi pr)

Reply:
ok

Reply:
thanks @gunzip, @chiadav please create a new PR for fixing the typos, thanks